src/store/boardStore.ts

```
@@ -1,380 +1,482 @@
+
+  subscribeToBoards: () => {
+    set({ isLoading: true, error: null })
+
+    const useFirebase = isFirebaseEnabled && db && !get().forceOfflineMode
+    if (useFirebase) {
+      const q = query(collection(db!, 'boards'), orderBy('createdAt'))
+
+      const unsubscribe = onSnapshot(
+        q,
+        async (snapshot) => {
+          let boards: Board[] = snapshot.docs.map(doc => {
```

ESLint

[error] 'boards' is never reassigned. Use 'const' instead.

(prefer-const)